webview/src/shared/variables.scss

```diff
@@ -17,6 +17,7 @@ $error-color: var(--vscode-errorForeground);
 $meta-cell-color: var(--vscode-descriptionForeground);
 $accent-color: var(--button-primary-background);
 $accent-color-transparent: var(--accent-transparency-3);
+$passed-color: var(--vscode-testing-iconPassed);
```

Comment:
Is there a better variable for this color that I missed?

Reply:
this is fine